File: HackerRank Challenge Solutions/Day 5/Solution.java
  import java.io.*;
  import java.util.*;
  import java.text.*;
  import java.math.*;
  import java.util.regex.*;

  public class Solution {

      public static void main(String[] args) {
          Scanner sc = new Scanner(System.in);
          int testCaseNum = sc.nextInt();
          for (int i = 0; i < testCaseNum; i++) {
              int a = sc.nextInt();
              int b = sc.nextInt();
              int n = sc.nextInt();
              int num = a;
              for (int j = 0; j < n; j++) {
                  num += b*Math.pow(2, j);
                  System.out.print(num + " ");
              }
              System.out.println();
          }
      }
  }

File: HackerRank Challenge Solutions/Day 5/Solution.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main(){
	int testCaseNum, a, b, num, n;
  	cin>>testCaseNum;
  	for (int i = 0; i < testCaseNum; i++) {
    	cin>>a>>b>>n;
		num=a;
        for (int j = 0; j < n; j++) {
        	num += b*pow(2, j);
            cout<<num<<" ";
        }
        cout<<endl;
    }
    return 0;
}
